cpg_infra/abstraction/base.py

```diff
@@ -355,3 +364,12 @@ def give_member_ability_to_list_buckets(
         self, resource_key: str, member, project: str = None
     ):
         return f'{resource_key} :: {member} can list buckets'
+
+    def bucket_output_path(self, bucket):
+        return f'Fake://{bucket}'
```

Comment:
Should this raise an exception instead, as long as this is not properly implemented?

Reply:
This is just a DEV CloudInfra implementation, useful for very quickly seeing which resources are created.